app/mixins/editor-base-controller.js

```diff
@@ -331,7 +331,16 @@ export default Mixin.create({
 
         notifications.showAlert(message, {type: 'error', delayed: delay, key: 'post.save'});
     },
-
+    keyDown: task(function* (event) {
+        switch (event.keyCode) {
+            // if the enter key or down key are pressed then focus on the editor
+            case 13:
+            case 40:
+                Ember.$('.__mobiledoc-editor').focus();
```

Comment:
I should add it's triggered by the keyDown action on gh-trim-focus-input within edit.hbs